tests/test_evaluation.py

```diff
@@ -35,64 +34,40 @@ def pred():
         }
     )
 
-    return pred
-
 
-def test_get_mspe(data, pred):
+def test_score_df(data, pred):
     """
     Return the expected forecast start, end and correct MSPE.
     """
     data = iup.IncidentUptakeData(data)
     pred = iup.PointForecast(pred)
 
-    output = iup.get_mspe(data, pred)
+    output = iup.score(data=data, pred=pred, score_fun=iup.mspe)
+    assert output.item(0, "forecast_start") == date(2020, 1, 1)
+    assert output.item(0, "forecast_end") == date(2020, 1, 5)
+    # we're not testing the actual value, just that we get some value
+    assert isinstance(output["score"][0], float)
 
-    output_date = output.select("forecast_start", "forecast_end")
 
-    assert output_date.equals(
-        pl.DataFrame(
-            {"forecast_start": date(2020, 1, 1), "forecast_end": date(2020, 1, 5)}
-        )
-    )
+def test_mspe():
+    x = np.array([0.0, 0.1, 0.7, 0.4, 0.5])
+    y = np.array([0.0, 0.2, 1.0, 0.6, 0.5])
+    assert np.isclose(iup.mspe(x, y), 0.028)
```

Comment:
Haha `isclose` is new to me. Should be useful in some of the other model-building tests, too. Python has everything.